app/uk/gov/hmrc/payetaxcalculatorfrontend/quickmodel/YouHaveToldUsItem.scala

```diff
@@ -78,7 +78,13 @@ object YouHaveToldUs {
     def toYouHaveToldUsItem(detail: Detail): YouHaveToldUsItem = {
       val label = s"${detail.period.replace(" ","_")}_sub"
       val idSuffix = "salary_period"
-      val url = routes.QuickCalcController.showSalaryForm().url
+      val url = {
+        detail.period match {
+          case "a day" => routes.QuickCalcController.showDaysAWeek(detail.amount, detail.url).url
```

Comment:
I don't like this String. Here should be a enumeration. This needs to be refactored globally, but later. Merging it now.